web-admin/src/features/alerts/metadata/AlertMetadata.svelte

```diff
@@ -62,10 +63,16 @@
     ? formatRefreshSchedule(alertSpec.refreshSchedule.cron)
     : "Whenever your data refreshes";
 
-  $: metricsViewAggregationRequest = JSON.parse(
+  $: queryArgsJson =
     (alertSpec?.resolverProperties?.query_args_json as string) ||
-      alertSpec?.queryArgsJson ||
-      "{}",
+    alertSpec?.queryArgsJson ||
+    "{}";
+  $: queryName =
+    alertSpec?.queryName ||
+    (alertSpec?.resolverProperties?.query_name as string) ||
+    "";
```

Comment:
It feels wrong for the `queryName` to be an empty string. Maybe `null` would be more semantically clear?